enhancements/update/available-update-metadata.md

```diff
@@ -0,0 +1,272 @@
+Exposing that information to in-cluster consumers allows them to use live metadata, instead of talking to Cincinnati directly or hard-coding expected values.
+For example, the web console currently [hard-codes an expected set of available channels][console-channels] for a given release; with this enhancement they could look up available channels in `status.desired.additionalMetadata["io.openshift.upgrades.graph.release.channels"]`.
+
+### Goals
+
+* Exposing release metadata provided by upstream Cincinnati for the currently-desired and available-update releases.
+* Documenting some well-known metadata keys.
+
+### Non-Goals
+
+* Restricting metadata keys to those documented in this enhancement.
+    Cincinnati may continue to add and remove keys as they see fit as long as they provide at least the keys required by this enhancement.
+* Teaching Cincinnati to support requests without `channel` query parameters.
+    This might be useful for recovering from users specifying [`desiredUpdate`][api-desiredUpdate] that were not in their configured [`channel`][api-channel], but I am fine leaving recovery to the users.
+    They should have received sufficient information to adjust their channel from wherever they received the target release they are supplying.
+
+## Proposal
+
+As proposed in [this API pull request][api-pull-request], this enhancement adds a new type:
+
+```go
+// Release represents an OpenShift release image and associated metadata.
+// +k8s:deepcopy-gen=true
+type Release struct {
+  // version is a semantic versioning identifying the update version. When this
+  // field is part of spec, version is optional if image is specified.
+  Version string `json:"version"`
+
+  // image is a container image location that contains the update. When this
+  // field is part of spec, image is optional if version is specified and the
+  // availableUpdates field contains a matching version.
+  Image string `json:"image"`
+
+  // Metadata is metadata associated with the release, such
+  // as associated channels (under
+  // io.openshift.upgrades.graph.release.channels) or a landing page
+  // (under url).  This information is extracted from within the
+  // release image, so it is covered by the same release image
+  // signatures used to verify the image.
+  //
+  // +optional
+  Metadata map[string]string `json:"metadata,omitempty"`
+
+  // additionalMetadata is additional metadata associated with the
+  // release.  Unlike the metadata property, the additionalMetadata
+  // property contains metadata that is not covered by the release
+  // image signatures.  For example, it may contain associated
+  // channels declared by the upstream update recommendation service.
+  // This information is as reliable as the upstream and connection used
+  // to retrieve it.
+  //
+  // Consumers who require static verification or
+  // non-repudiation should avoid additionalMetadata and prefer metadata.
```

Comment:
You should pick a name that indicates the metadata is untrusted.  `externalMetadata` probably.  I don't think you should use `untrustedMetadata` (as accurate as it is) because it's not untrusted, it's trusted differently.  Maybe `upstreamMetadata` (i like that better than `externalMetadata`).

Reply:
Pivoted from `additionalMetadata` to `upstreamMetadata` with 486fd17 -> 5b48492.